3-0-spring-mvc/src/main/resources/templates/notes.html

```diff
@@ -0,0 +1,44 @@
+<!doctype html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+HK:wght@100;400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" th:href="@{css/style.css}" href="../static/css/style.css">
+    <script th:src="@{/js/jquery-3.5.1.js}" src="../static/js/jquery-3.5.1.js"></script>
+    <title>Notes</title>
+</head>
+<body>
+<div class="wrapper">
```

Comment:
@erneman1 overall the page looks very nice. I was not able to add a note, we can discuss it later. And regarding the design, it would be better to have a form on the botton. E.g. you have a list of your notes, and then input on the bottom...